text/0997-make-on-built-in.md

```diff
@@ -0,0 +1,136 @@
+---
+stage: accepted
+start-date: 2023-12-22T00:00:00.000Z
+release-date: # In format YYYY-MM-DDT00:00:00.000Z
+release-versions:
+teams: # delete teams that aren't relevant
+  - framework
+  - learning
+  - typescript
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/997
+project-link:
+suite: 
+---
+
+<!--- 
+Directions for above: 
+
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
+release-date: Leave as is
+release-versions: Leave as is
+teams: Include only the [team(s)](README.md#relevant-teams) for which this RFC applies
+prs:
+  accepted: Fill this in with the URL for the Proposal RFC PR
+project-link: Leave as is
+suite: Leave as is
+-->
+
+# Make `{{on}}` a built in modifier  
+
+## Summary
+
+Today, when using gjs/gts/`<template>`, in order to bind event listeners, folks _must import_ the `{{on}}` modifier.
```

Comment:
This is not true as discussed in #1007. Folks could also use the `on*` properties of the element.

Reply:
Given the discovery of `onclick`'s usefulness for the use case that I'm mostly thinking about with this RFC, I think _I_ may even need convincing for this RFC to be pursued.

Related decision chart: https://github.com/emberjs/rfcs/pull/1007#issuecomment-1950275078

Reply:
yes but that has nothing to do with this proposal, you can always use normal html attributes

Reply:
I just mean, that because of `onclick` (and similar), I'm not sure we need This (RFC #997) at all

Reply:
> you can always use normal html attributes

My main concerns with `<button onclick={{foo}}>` is that it looks as setting attributes while actually setting a property.

But I fear that's a little bit off topic here. 🙃